perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py

```diff
@@ -344,12 +345,12 @@ def _perform_testing(dir, test_type, num_samples):
     # Creating config file for mounting with hns enabled.
     with open("/tmp/config.yml",'w') as mount_config:
       mount_config.write("enable-hns: true")
-    mount_flags="--config-file=/tmp/config.yml --stackdriver-export-interval=30s"
+    mount_flags="--config-file=/tmp/config.yml --log-severity=trace --log-format \"text\" --log-file {} --stackdriver-export-interval=30s".format(LOG_FILE)
```

Comment:
Looks like this line hasn't been formatted properly. It should have had a space before and after the `=`.

Reply:
Obsolete Code.